tests/Metadata/MetaDataBaseTest.php

```diff
@@ -315,19 +315,19 @@ protected function nestedChange(array $keys, array &$data, $newValue) : void
      */
     public function providerExpires() : array
     {
-        return [
+        return static::getKeyedArray([
             ['1970', false],
-            ['1970-01-01T00:00:00Z', false],
+            ['1970-01-01T00:00:00Z', true],
```

Comment:
A couple test cases now pass because we don't have a hard-code "2" to start the year. Think that is fine.

Just means our descents won't have have mad rush to fix this is 2999 😉